### ISSUE TITLE
Bounce

### ISSUE THREAD

**Issue by frasertweedale** [purebred-mua/purebred#315]
**Is your feature request related to a problem? Please describe.**
Need a way to bounce mail, i.e. to re-send a message as-is but explicitly specify SMTP recipient(s)

**Describe the solution you'd like**
No need for a compose screen, but we will need to specify recipient(s) somehow.

**Describe alternatives you've considered**
Retransmit via carrier pigeon.

**Additional context**
`()`

**Comment by frasertweedale** [comment]
Unsure if should be v1 or not.  It sounds not too hard though.

**Comment by romanofski** [comment]
Yeah not sure. I can't live without forward, but bounce probably. Although I've used bouncing of mails a fair bit.

**Comment by frasertweedale** [comment]
OK, make it... v1.1 then? :)
